index.html

```diff
@@ -211,7 +211,7 @@ <h6 class="is-6 title">Sixth level</h6>
     </div>
   </section>
 
-  <section class="hero has-background-grey-light" aria-labelled="features-section">
+  <section class="hero has-background-grey-light" aria-label="features-section">
```

Comment:
Could you please add `aria-hidden="true"`

<img width="248" alt="image" src="https://user-images.githubusercontent.com/16785774/172824412-80f04963-99c1-48ee-b67b-2e6e5321ec9e.png">

<img width="297" alt="image" src="https://user-images.githubusercontent.com/16785774/172824624-6351b39c-51bf-4edd-b47d-70cc08eda8eb.png">

<img width="443" alt="image" src="https://user-images.githubusercontent.com/16785774/172824858-325f47cd-0e76-4abf-b08a-4ad6941c2d79.png">

<img width="280" alt="image" src="https://user-images.githubusercontent.com/16785774/172824981-d6703334-0853-4882-b373-21c1650be058.png">

Reply:
Sorry. What is wrong with `aria-hidden="true"` at 282, 288, and 294 lines?

Reply:
oops, the lines 282, 288, and 294 - are correct ))